app/components/participant-creator/template.hbs

```diff
@@ -76,21 +70,18 @@
     <div class="col-md-4">
         <p>
             <strong>
-                Preview:
+                Example:
```

Comment:
Thanks! Since the data in this table is thrown away, clarifying that it's not real data is a really helpful UI improvement!